### PR TITLE
Implemented following code to resolve the URL issue in the Framework, CustomControl.,ProjectX_sample application at VS2013 VB

### PR DESCRIPTION
**Pull request by supragyan**
Resolved URL issue in the Framework and CustomControl component.  
 Resolved URL issue in ProjectX_Sample.

### CODE REVIEW

File: root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/testBlankScreen.master [hunk 1]
```diff
@@ -19,7 +24,8 @@
     <!-- IE8をIE7互換モードで動作させる -->
     <!--meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" /-->
     
-    <link rel="stylesheet" href="/ProjectX_sample/Css/style.css" type="text/css"/>
+    <link rel="stylesheet"  type="text/css" href="<%= Me.ResolveClientUrl("~/Css/style.css") %>">/>
```
Comment (yusukemaegawa) [inline root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/testBlankScreen.master]: @supragyan 

"/>" at the end is unnecessary.
```html
<link rel="stylesheet"  type="text/css" href="<%= Me.ResolveClientUrl("~/Css/style.css") %>">
```

Comment (daisukenishino) [inline root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/testBlankScreen.master]: @yusukemaegawa 

I was surprised that you can notice slight changes well.

Comment (SandeepNayak077) [inline root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/testBlankScreen.master]: @yusukemaegawa 

Thank you for identifying this minute error.
We are closing this pull request once @yusukemaegawa review is completed.

Please notify us if there is any other changes.

Comment (yusukemaegawa) [inline root_VS2013/programs/VB/Samples/WebApp_sample/ProjectX_sample/Aspx/Common/testBlankScreen.master]: @daisukenishino 

>I was surprised that you can notice slight changes well.

You can see "/>" in following image.

screencapture

**Comment by SandeepNayak077** [comment]
@daisukenishino 
@yusukemaegawa 

I have closed this pull request and requested @supragyan to create new pull request with @yusukemaegawa's comment incorporation.

**Pull request closed without merging**